modules/storage.js

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-unused-vars */
 class Storage {
   static getBooks() {
```

Comment:
- [ ] it would be better if you use the es6 `arrow function` instead of the `normal function` in order to practice to work with both